src/main/resources/io/github/dfdlschemas/envelopepayload/xsd/envelopePayload.dfdl.xsd

```diff
@@ -1,98 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This schema with no namespace replaces the one with this name, but now it defines the root element only.